lib/y2_installbase.pm

```diff
@@ -187,6 +187,42 @@ sub save_upload_y2logs {
     $self->investigate_yast2_failure() unless $args{skip_logs_investigation};
 }
 
+sub save_remote_upload_y2logs {
+    my ($self, %args) = @_;
+
+    return if (get_var('NOLOGS'));
+    $args{suffix} //= '';
+
+    # Do not test/recover network if collect from installation system, as it won't work anyway with current approach
+    # Do not recover network on non-qemu backend, as not implemented yet
+    $args{no_ntwrk_recovery} //= (get_var('BACKEND') !~ /qemu/);
+
+    # Try to recover network if cannot reach gw and upload logs if everything works
+    if (can_upload_logs() || (!$args{no_ntwrk_recovery} && recover_network())) {
+        type_string 'sed -i \'s/^tar \(.*$\)/tar --warning=no-file-changed -\1 || true/\' /usr/sbin/save_y2logs';
+        send_key 'ret';
+        my $filename = "/tmp/y2logs$args{suffix}.tar" . get_available_compression();
+        type_string "save_y2logs $filename\n";
+        upload_logs $filename;
+    } else {
+        # Redirect logs content to serial
+        wait_screen_change { type_string("journalctl -b --no-pager > /dev/$serialdev") };
+        send_key('ret');
+        wait_still_screen 5;
+        wait_screen_change { type_string("dmesg > /dev/$serialdev") };
+        send_key('ret');
+        wait_still_screen 5;
+        wait_screen_change { type_string("cat /var/log/YaST2/y2log > /dev/$serialdev") };
+        send_key('ret');
+        wait_still_screen 5;
+        type_string("cat /var/log/YaST2/y2log\n");
+    }
+    save_screenshot();
+    # We skip parsing yast2 logs in each installation scenario, but only if
+    # test has failed or we want to explicitly identify failures
+    $self->investigate_yast2_failure() unless $args{skip_logs_investigation};
```

Comment:
I doubt that this method can work in case of remote installations as uses `script_output` which requires serial code, so, please, just remove this option.

Reply:
done